server/src/main/java/fr/uga/l3miage/tp1/exo2/models/SessionEntity.java

```diff
@@ -0,0 +1,17 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@Entity
+public class SessionEntity {
+    @Id
+    private UUID idSession;
+    private String lasCommand;
+    private String currentDir;
+    private Boolean lock;
```

Comment:
```suggestion
    private boolean lock;
```
attention `Boolean` et `boolean` sont différent, l'un est un objet, et peux être `null`, alors que l'autre est un type primitif, donc ne peux jamais être nul